### DISCUSSION TITLE
Connecting AD with wiki-js, self-registration of users

### DISCUSSION THREAD

**Discussion by papagaya** [requarks/wiki#7336] [General]
Hello,

I have a "LDAP / Active Directory" connection in my Wiki.js 2.5.303.

After the configuration, I couldn't log in automatically with my account, so I added myself manually to Wiki.js.

Can someone please tell me how to configure the self-registration in "LDAP / Active Directory" from the AD.

The self-registration in "Local" works perfectly.

what does the registration design look like when you log in for the first time?

Thank you in advance

Best regards

**Comment by heraldodinero** [comment]
Have you read through this? I did the exact same thing as you and this document did the job 100% for me:

https://docs.requarks.io/auth/ldap

**Reply by papagaya** [reply]
The LDAP function works perfectly for me. However, I have to add the users manually and cannot see the registration button.
can you see the registrations button on the Login interface over LDAP ?

**Reply by fnafox** [reply]
There is no registration button for LDAP as it's expected that you create and manage users in your AD. Self registration in LDAP context functions by simply admitting authorized users and assigning them to a group if set up to do so.
